_topic_maps/_topic_map.yml

```diff
@@ -3570,8 +3570,6 @@ Topics:
     File: virt-migrate-vmi
   - Name: Migrating a virtual machine over a dedicated additional network
     File: virt-migrating-vm-on-secondary-network
-  - Name: Monitoring live migration
-    File: virt-monitor-vmi-migration
```

Comment:
I understand the rationale for moving the monitoring live migration assembly to the monitoring directory. At the same time we should probably keep some info in the live migration procedures. I don't want to hold up your PR by requesting changes to the migration assembly. The live migration assembly names are a bit weird, so I will try to create a separate task to improve the migration section. So this is just an FYI.